test/e2e/grpc_test.go

```diff
@@ -317,15 +317,23 @@ func streamTest(tc *TestContext, host, domain string) {
 
 func testGRPC(t *testing.T, f grpcTest, fopts ...rtesting.ServiceOption) {
 	t.Helper()
-	t.Parallel()
+	if !test.ServingFlags.HTTPS {
+		t.Parallel()
+	}
 
 	// Setup
 	clients := Setup(t)
 
 	t.Log("Creating service for grpc-ping")
 
+	svcName := test.ObjectNameForTest(t)
+	// Long name hits this issue https://github.com/knative-sandbox/net-certmanager/issues/214
+	if t.Name() == "TestGRPCStreamingPingViaActivator" {
```

Comment:
This very specific check makes me somewhat uncomfortable. Can we do something more generic? Do we need to make sure that the **entire domain** is < 64 bytes?

Reply:
Yes, we need to make sure that the entire domain is 64 bytes...
So, I don't have good idea to make it generic. I updated but it would not be good code I think :sweat:

Reply:
I reverted my change. I think that the current specific workaround is simple and better. (But please let me know if you have a better idea :sweat_smile: )